pkg/tls/codec.go

```diff
@@ -105,7 +105,7 @@ func DecodePEMCertificates(txt string) (certs []*x509.Certificate, err error) {
 	return
 }
 
-// CertificatesToPool covererts a slice of certificates into a cert pool
+// CertificatesToPool coverts a slice of certificates into a cert pool
```

Comment:
I think this should be "converts"

Reply:
Indeed. I think I had that in mind at one point but then was careless in copying a correction.